app/models/sir_trevor_rails/block.rb

```diff
@@ -18,12 +18,20 @@ def format
       send(:[], :format).present? ? send(:[], :format).to_sym : DEFAULT_FORMAT
     end
 
+    def self.alt_text?
```

Comment:
Is there a reason this is defined as a class method and not an instance method? You're calling it here https://github.com/projectblacklight/spotlight/pull/3264/files#diff-50de7f3fa74c59817c7b61c1adf10178b1039ba6b7c95529962359aef2a4f13fR18 as if it were an instance method.

Reply:
Yeah these lines need it to be a class method: https://github.com/projectblacklight/spotlight/pull/3264/files#diff-9da975442affa1f40c188c0b08c324ac361b8b3e51d484e78d5633de76dc92e1R31-R34

Reply:
Which I now realize you need both. It was an instance until I added the line above. I have updated the model to call the classes' variable for the instance https://github.com/projectblacklight/spotlight/pull/3264/files#diff-9da975442affa1f40c188c0b08c324ac361b8b3e51d484e78d5633de76dc92e1R21-R23